public/index.html

```diff
@@ -59,5 +59,14 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <ion-icon name="search-sharp"></ion-icon>
```

Comment:
The icons should be added in JSX

Reply:
ionicons are web components. See https://developers.google.com/web/fundamentals/web-components/customelements.